scripts/eurostat/birth_death_migration/import_data.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+
+
+def download_data():
+    """Downloads raw data from Eurostat website and stores it in instance
+    data frame.
+    """
+    raw_df = pd.read_table(DATA_LINK)
+    return raw_df
+
+
+def preprocess_data(raw_df):
+    """Preprocesses instance raw_df and puts it into long format."""
+    if raw_df is None:
+        raise ValueError("Uninitialized value of raw data frame. Please "
```

Comment:
This error message seems to assume context of the entire script. Instead, you might raise a ValueError simply stating that input `raw_df` should not be None?

Reply:
Could even just be an assert like below.

Reply:
decided to remove it since it is not very meaningful